README.md

```diff
@@ -143,6 +143,37 @@ index 2878332..1f3103a 100644
 +CMD [ "/usr/bin/python3" ]
 ```
 
+# How to use Claude AI
+
+To access Claude AI, you need an API key. Follow these steps to get started:
+
+1. Create an account
+- Visit [Anthropic Console](https://console.anthropic.com/login) and sign up with your e-mail. You'll recieve a sign in link to the Anthropic Console. 
+
+<img src="/assets/anthropic_signup.png" width="90%">
+
+2. Get your API key
+- Once logged in, navigate to **Get API keys**.
+- Purchase the amount of credits you need. After completing the purchase, you'll be able to generate the key through the API console.
+
+<img src="/assets/anthropic_api_payment.png" width="90%">
+
+3. Log in with your API key
+- Navigate to your WebVM and click the robot icon to open the Claude AI Integration tab.
```

Comment:
Incorrect, you don't have to click and clicking actually closes the tab. This is behavior you have implemented @AtibQur you should be aware of it